client/my-sites/theme/main.jsx

```diff
@@ -813,12 +829,14 @@ class ThemeSheet extends Component {
 				>
 					{ ! retired && ! hasWpOrgThemeUpsellBanner && ! isWPForTeamsSite && this.renderButton() }
 				</HeaderCake>
-				<div className="theme__sheet-columns">
+				<div className={ columnsClassName }>
 					<div className="theme__sheet-column-left">
 						{ ! retired && this.renderSectionContent( section ) }
 						{ retired && this.renderRetired() }
 					</div>
-					<div className="theme__sheet-column-right">{ this.renderScreenshot() }</div>
+					{ ! this.isRemoved() && (
```

Comment:
Super nitpicky, on this line and line 801 we're calling the `isRemoved` method, we could call it once and store the result.